File: C/1061.c
#include <stdio.h>
int main() {
    int dia1, dia2, hora1, hora2, minuto1, minuto2, segundo1, segundo2, tempo1, tempo2, intervalo, acrescimo=0, acrescimo2=0, acrescimo3=0;

    scanf("Dia %d", &dia1);
    scanf("%d : %d : %d", &hora1, &minuto1, &segundo1);
    
    scanf("\nDia %d", &dia2);
    scanf("%d : %d : %d", &hora2, &minuto2, &segundo2);


    if (hora1==0)
        hora1=24;
    if (hora2==0)
        hora2=24;

    if (segundo2>=segundo1 && minuto2!=minuto1) segundo2=segundo2-segundo1;
    else {
        segundo2=60-segundo1+segundo2;
        acrescimo2=1;
    }

    if (minuto1==minuto2 &&segundo2!=0) {
        minuto2=59;
        acrescimo=1;
    }
    else if (minuto2>=minuto1) minuto2=minuto2-minuto1-acrescimo2;
    else if (segundo2<segundo1) {
        minuto2=(60-(minuto1-minuto2))-acrescimo2;
        acrescimo=1;}
    else {
        minuto2=60-minuto1+minuto2;
        acrescimo=1;
    }

    if (hora2>hora1)
        hora2=hora2-hora1-acrescimo;
    else if (hora2==hora1 && minuto2>minuto1)
        hora2=0;
    else if (hora2==hora1 && minuto2<=minuto1)
        hora2=24-acrescimo;
    else if (hora2<hora1) {
        hora2=(24-(hora1-hora2))-acrescimo;
        acrescimo3=1;
        }

    if (segundo2>=60) {
        segundo2-=60;
        minuto2++;
    }
    if (minuto2>=60) {
        minuto2-=60;
//        hora2++;
    }
    if (hora2>=24) {
        hora2-=24;
        dia2++;
    }
    

    printf("%d dia(s)\n", dia2 -(dia1+acrescimo3));
    printf("%d hora(s)\n", hora2);
    printf("%d minuto(s)\n", minuto2);
    printf("%d segundo(s)\n", segundo2);

    return 0;
}

File: Python/1061.py
me=0
he=0
de=0

linha1=input().split(" ")
dia1=int(linha1[1])
linha2=input().split(" ")
h1=int(linha2[0])
m1=int(linha2[2])
s1=int(linha2[4])

linha3=input().split(" ")
dia2=int(linha3[1])
linha4=input().split(" ")
h2=int(linha4[0])
m2=int(linha4[2])
s2=int(linha4[4])

if (s2>=s1): sr=s2-s1
elif (s2<s1):
    sr=60-s1+s2
    me=1

if (m2>=me+m1): mr=m2-(m1+me)
elif (m2<me+m1):
    mr=60-(m1+me)+m2
    he=1

if (h2>=he+h1): hr=h2-(h1+he)
elif (h2<he+h1):
    hr=24-(h1+he)+h2
    de=1
    
dr=dia2-(dia1+de)

print("{} dia(s)" .format(dr))
print("{} hora(s)" .format(hr))
print("{} minuto(s)" .format(mr))
print("{} segundo(s)" .format(sr))
